### PR TITLE
Feature/sync with 1.0.1 bumped polkadotjs

### PR DESCRIPTION
**Pull request by Raid5594**
- Update polkadot js version to be compatible with upgraded nodes
- Update filter settings for validator for test networks
- Added DEV / QANET testnets to the codebase

### CODE REVIEW

File: src/locale/cn/help.json [hunk 1]
```diff
@@ -299,7 +299,7 @@
     "externals": {
       "connect_your_accounts": "如何连接您的帐户",
       "how_to_use": "如何使用Staking Dashboard:概述",
-      "stake_dot": "抵押您的DOT",
+      "stake_cere": "抵押您的DOT",
```
Comment (MRamanenkau) [inline src/locale/cn/help.json]: Minor: should be 抵押您的CERE

File: src/library/GenerateNominations/useFetchMethods.tsx [hunk 2]
```diff
@@ -129,7 +137,10 @@ export const useFetchMehods = () => {
   };
 
   const available = (nominations: any) => {
+    console.log(`Validators to be available for nominations:`);
+    console.log(nominations);
     const _nominations = Object.assign(validators);
+    console.log(_nominations);
```
Comment (MRamanenkau) [inline src/library/GenerateNominations/useFetchMethods.tsx]: Why do we need it? Leaving `console.logs` usually considered as bad practice in frontend apps

**Comment by MRamanenkau** [comment]
> * Update polkadot js version to be compatible with upgraded nodes
> * Update filter settings for validator for test networks
> * Added DEV / QANET testnets to the codebase

1. Please specify what version of Substrate is compatible with this update
2. Check comments [here](https://github.com/Cerebellum-Network/staking-dashboard/commit/ddb02f454a37d4fea68c0de5df50ba14d195f968)

File: src/library/GenerateNominations/useFetchMethods.tsx (hunk 2 above)
Comment (Raid5594) [reply src/library/GenerateNominations/useFetchMethods.tsx]: It's a mistake, probably didn't commit the file

File: src/locale/cn/help.json (hunk 1 above)
Comment (Raid5594) [reply src/locale/cn/help.json]: nice

Comment (Raid5594) [reply src/locale/cn/help.json]: done

File: src/library/GenerateNominations/useFetchMethods.tsx (hunk 2 above)
Comment (Raid5594) [reply src/library/GenerateNominations/useFetchMethods.tsx]: done

**Review by MRamanenkau — APPROVED** [review]

**Pull request merged**